README.md

```diff
@@ -19,47 +19,47 @@ Join us on our [public Slack channel](https://slack.textile.io/) for news, discu
-  * [Sharing bucket files and folders](#sharing-bucket-files-and-folders)
-  * [Creating a Filecoin bucket archive](#creating-a-filecoin-bucket-archive)
-  * [Multi-writer buckets](#multi-writer-buckets)
-  * [Deleting a bucket](#deleting-a-bucket)
-  * [Using the Buckets Library](#using-the-buckets-library)
-    * [Creating a bucket](#creating-a-bucket-1)
-    * [Getting an existing bucket](#getting-an-existing-bucket)
-    * [Pushing local changes](#pushing-local-changes)
-    * [Pulling remote changes](#pulling-remote-changes)
-  * [Using the Mail Library](#using-the-mail-library)
-    * [Creating a mailbox](#creating-a-mailbox)
-    * [Getting an existing mailbox](#getting-an-existing-mailbox)
-    * [Sending a message](#sending-a-message)
-    * [Watching for new messages](#watching-for-new-messages)
-* [Developing](#developing)
-* [Contributing](#contributing)
-* [Changelog](#changelog)
-* [License](#license)
+- [textile](#textile)
+  - [Security](#security)
+  - [Background](#background)
+  - [Install](#install)
+    - [The Hub](#the-hub)
+      - [`hubd`](#hubd)
+      - [`hub`](#hub)
+    - [Buckets](#buckets)
+      - [`buckd`](#buckd)
+      - [`buck`](#buck)
+    - [The Buckets Library](#the-buckets-library)
+  - [Getting Started](#getting-started)
+    - [The Hub](#the-hub-1)
+    - [Running Buckets](#running-buckets)
+    - [Creating a bucket](#creating-a-bucket)
+    - [Creating a private bucket](#creating-a-private-bucket)
+    - [Adding files and folders to a bucket](#adding-files-and-folders-to-a-bucket)
+    - [Recreating an existing bucket](#recreating-an-existing-bucket)
+    - [Creating a bucket from an existing Cid](#creating-a-bucket-from-an-existing-cid)
+    - [Exploring bucket contents](#exploring-bucket-contents)
+    - [Resetting bucket contents](#resetting-bucket-contents)
+    - [Watching a bucket for changes](#watching-a-bucket-for-changes)
+    - [Protecting a file with a password](#protecting-a-file-with-a-password)
+    - [Sharing bucket files and folders](#sharing-bucket-files-and-folders)
+    - [Creating a Filecoin bucket archive](#creating-a-filecoin-bucket-archive)
+    - [Multi-writer buckets](#multi-writer-buckets)
+    - [Deleting a bucket](#deleting-a-bucket)
+    - [Using the Buckets Library](#using-the-buckets-library)
+      - [Creating a bucket](#creating-a-bucket-1)
+      - [Getting an existing bucket](#getting-an-existing-bucket)
+      - [Pushing local files](#pushing-local-files)
+      - [Pulling remote changes](#pulling-remote-changes)
+    - [Using the Mail Library](#using-the-mail-library)
+      - [Creating a mailbox](#creating-a-mailbox)
+      - [Getting an existing mailbox](#getting-an-existing-mailbox)
+      - [Sending a message](#sending-a-message)
+      - [Watching for new messages](#watching-for-new-messages)
+  - [Developing](#developing)
+  - [Contributing](#contributing)
+  - [Changelog](#changelog)
+  - [License](#license)
```

Comment:
ugh... sorry if my lint here is dif than the rest of you guys. will PR back if i'm off, just lmk